addons/mail/i18n/mail.pot

```diff
@@ -3879,7 +3879,7 @@ msgstr ""
 #. openerp-web
 #: code:addons/mail/static/src/components/welcome_view/welcome_view.xml:0
 #, python-format
-msgid "Logged as"
+msgid "Logged in as"
```

Comment:
If we change the source term we might as well make it properly translatable with the name inside.